config/static_test.go

```diff
@@ -35,9 +37,8 @@ BlackListed:
 Beacon:
     DefaultConnectionThresh: 24
 Filtering:
-    AlwaysInclude: ["8.8.8.8/32"]
-    InternalSubnets: ["10.0.0.0/8","172.16.0.0/12","192.168.0.0/16"]
-
+    AlwaysInclude: [8.8.8.8/32]
+    InternalSubnets: [10.0.0.0/8,172.16.0.0/12,192.168.0.0/16]
```

Comment:
I was just testing to see if the quotes were necessary and forgot to put them back in.  They are apparently not needed but I can add them back if we want.